labels.mss

```diff
@@ -533,12 +533,14 @@
      marker-max-error: 0.5;
      marker-spacing: 100;
      marker-fill: #777777;
-     [oneway != 'yes'][oneway_bicycle = 'yes'] {
-       marker-fill: #00f;
-     }
-     [highway='cycleway'] {
+     [highway='cycleway'],
+     [highway='path'] {
        marker-fill: #ddf;
      }
+     /* Weird case where the road is oneway for only the bicycles, blue arrow. */
+     [oneway != 'yes'][oneway_bicycle = 'yes'][highway!='path'][highway!='cycleway'] {
```

Comment:
This can simply be `[oneway != 'yes'][oneway_bicycle = 'yes']` if it is put before the previous `[highway='cycleway'], [highway='path']` rule above. Overloading should do the trick.

Why is it a "weird case"? This is just any countraflow bike lane, isn't it?

Reply:
No, it is a two way road, but one way for bicycle, weird, may be a bug, may be we should not render it.

Reply:
0.7% of cases

Reply:
Few examples around https://www.openstreetmap.org/way/380258015, https://www.openstreetmap.org/way/679211980, https://www.openstreetmap.org/way/650712060

For me, this is a tagging error and should not be taken into account. I'll double check.

Reply:
Note that if we consider this case, we should consider the symmetric case also `[oneway != '-1'][oneway_bicycle = '-1']` (any two-way street for cars which is one-way only for bicycles).

Reply:
Your examples looks like tagging error, though the theoric case can exist in real life.
But not sure if has at least an example in this world.
May be a cycle track in oneway along a bigroad forbiden for cyclist ?

Reply:
Since the use case is realy rare, we can remove it.